.github/ct.yaml

```diff
@@ -2,3 +2,4 @@ validate-maintainers: false
 check-version-increment: false
 chart-dirs:
   - helm
+target-branch: master
```

Comment:
Here, in the `maint-0.9` branch, the `target-branch` should be `maint-0.9`, isn't it? (And we keep it on `master` when merging upstream.)